commitizen/cli.py

```diff
@@ -92,6 +92,12 @@
                         "default": False,
                         "help": "generate the changelog for the newest version",
                     },
+                    {
+                        "name": ["--no-verify"],
+                        "action": "store_true",
+                        "default": False,
+                        "help": "disable git hooks during bumping",
```

Comment:
I just take a look of the documentation of git. The help message is 

```text
--no-verify
    This option bypasses the pre-commit and commit-msg hooks.
```

I think it might be better if we can align with them